package.json

```diff
@@ -36,7 +36,7 @@
     "bindings": "~1.3.0",
     "fast-future": "~1.0.2",
     "nan": "~2.10.0",
-    "prebuild-install": "^2.1.0"
+    "prebuild-install": "^4.0.0"
```

Comment:
We skipped 3.0.0. Not really relevant here, but was that major just for private repo support?

Reply:
Correct.